src/coreclr/jit/compiler.h

```diff
@@ -8418,253 +8385,21 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
     struct SIMDHandlesCache
     {
-        // BYTE, UBYTE, SHORT, USHORT, INT, UINT, LONG, ULONG
-        // NATIVEINT, NATIVEUINT, FLOAT, and DOUBLE
-        static const uint32_t SupportedTypeCount = 12;
-
-        // SIMD Types
-        CORINFO_CLASS_HANDLE VectorTHandles[SupportedTypeCount];
-
         CORINFO_CLASS_HANDLE PlaneHandle;
         CORINFO_CLASS_HANDLE QuaternionHandle;
         CORINFO_CLASS_HANDLE Vector2Handle;
         CORINFO_CLASS_HANDLE Vector3Handle;
         CORINFO_CLASS_HANDLE Vector4Handle;
         CORINFO_CLASS_HANDLE VectorHandle;
```

Comment:
Why do these ones need to stay?

Reply:
Is it just for `isOpaqueSIMDType`? Is there a plan to also remove that somehow?

Reply:
> Is it just for `isOpaqueSIMDType`?

Yep. No explicit plans for its removal currently, though one could certainly imagine rewriting it to not need the cached handles (e. g. a new `getClassAttribs` flag, or matching by name).